### ISSUE TITLE
The number of attempts is 1 more than "freeRetries" value

### ISSUE THREAD

**Issue by Tezaruz** [AdamPflug/express-brute#85]
Readme says that

> freeRetries The number of retries the user has before they need to start waiting (default: 2)

but if you install freeRetries === 5, then the library gives you 6 queries and only after that the user starts to wait.

**Comment by TheAndroidGuy** [comment]
Duplicate of #82
